community-membership.md

```diff
@@ -46,7 +46,7 @@ remain active contributors to the community.
       forums, Stack Overflow)
 - Subscribed to [kubernetes-dev@googlegroups.com]
 - Have read the [contributor guide]
-- Actively contributing to 1 or more subprojects.
+- Actively contributing non-trival pulls to 1 or more subprojects.
```

Comment:
I'm concerned that non-trivial is subjective. There is no strict codified definition of what is non-trivial and what isn't. While we can agree that typos are trivial, the line soon blurs down.

Like @cblecker  mentioned, I would also expect that if an existing member is sponsoring someone to become a member, they would only do so if they have had close interactions with the non-member and they think that the non-member's contributions are "non-trival enough according to them" for warranting membership.

Reply:
> I'm concerned that non-trivial is subjective.

indeed 😅at least we can start by typofix